stylesheets/blue/components/_nav.scss

```diff
@@ -209,7 +209,7 @@ $Nav-item-color-hover: $Theme-color-yellowOrange;
   margin-left: -$Theme-spacing-small;
 }
 
-@include media('>smallDesktop') {
+@include media('>900px') {
```

Comment:
I did, although I find we currently don't have a good overall responsive system, which makes all seem rather ad-hoc, so naming this didn't felt like an improvement